projects/arc/src/app/app-routing.module.ts

```diff
@@ -6,24 +6,24 @@ import {AuthGuard, LoggedInGuard} from '@project-lib/core/auth';
 const routes: Routes = [
   {
     path: 'auth',
-    loadChildren: () =>
-      import('projects/arc-lib/src/lib/components/auth/auth.module').then(
-        m => m.AuthModule,
-      ),
-    canActivate: [LoggedInGuard],
+  loadChildren: () =>
+  import('projects/arc-lib/src/lib/components/auth/auth.module').then(
+  m => m.AuthModule,
+  ),
+  // canActivate: [LoggedInGuard],
   },
   {
     path: 'main',
     loadChildren: () => import('./main/main.module').then(m => m.MainModule),
-    canActivate: [AuthGuard],
+    // canActivate: [AuthGuard],
```

Comment:
why commented while commiting
please recheck before pushing the code 
this is a wrong practice

Reply:
my mistake it pushed like that i ll take care